x-pack/filebeat/module/azure/signinlogs/_meta/fields.yml

```diff
@@ -163,4 +163,51 @@
           type: keyword
           description: >
             Status
-
+        - name: authentication_requirement_policies
+          type: keyword
+          description: >
+            Set of CA policies that apply to this sign-in, each as CA: policy name, and/or MFA: Per-user.
+        - name: applied_conditional_access_policies
+          type: object
+          description: >
+            Details of the conditional access policies being applied for the sign-in.
+        - name: resource_tenant_id
+          type: keyword
+          description: >
+            The resource tenantId for B2B(business-to-business) scenarios.
+        - name: authentication_details
+          type: object
+          description: >
+            A record of each step of authentication undertaken in the sign-in.
+        - name: authentication_processing_details
```

Comment:
Also similar to above. However, it looks like each entry just has two fields, `key`, and `value`, which are both strings. Wondering if maybe then we can use `flattened`, which is like `nested`, but without the total number of subdocument restrictions. See https://www.elastic.co/guide/en/elasticsearch/reference/current/flattened.html

Thoughts @andrewkroh ? I believe we do use `flattened` in a couple of integrations already?

Reply:
Done.

(Note: this is not a new field, just was missing from fields.yml)

Reply:
Unlike the `nested` comment above, I don't think that the query characteristics here will change, you can still use `terms`, `match`, etc. -- but I'm not super familiar with how switching types works when wildcarding across indices with different types (same idea with the `filebeat-*` query above) -- we may want to test it out.